cubano-core/src/main/java/org/concordion/cubano/driver/action/ActionWait.java

```diff
@@ -243,15 +243,19 @@ public int getAttempts() {
         attempts = 0;
 
         Instant start = clock.instant();
-        Instant end = null;
+        Instant end = clock.instant();
```

Comment:
Fixed

Reply:
I actually meant `Instant end = start;` to make sure they are identical. As it stands there'll be some difference between the 2 `Instant`s. Since `Instant` is immutable, you'll get a new `Instant` if you change it, so this will be safe.

Reply:
Thats what I did, this thread is showing the older version of the code without my changes - which I personally find rather confusing!  Took me ages to work out my commit had made it.